docs/iOS/readme_iOS.md

```diff
@@ -1,69 +1,51 @@
 
 You can follow the standard <a href="/readme.md#importing-the-plugin">Importing the Plugin</a> process. With a few changes here when <a href="#running-the-samples">running the samples</a> and <a href="#configuring-the-plugin">configuring the plugin</a>.
-> :heavy_exclamation_mark: If you choose the tarball method, when downloading the release on mac it may convert the ```.tgz``` into a ```.tar``` which is not compatible with unity. Changing the file extension back to a ```.tgz``` should fix this.
 
-<br />
+> [!WARNING]
+> If you choose the tarball method, when downloading the release on mac it may convert the `.tgz` into a `.tar` which is not compatible with unity. Changing the file extension back to a `.tgz` should fix this.
 
-## Samples
+## <div align="center">Samples</div>
 
 You can follow the standard <a href="/readme.md#samples">Samples</a> process. Please note the details in the <a href="#running-the-samples">Running the samples</a> section when running the samples from a build for iOS.
-> :heavy_exclamation_mark: The EOS Overlay is not implemented yet. But when it is due to the limitations of phones, the EOS Overlay is not set to be openable by a physical button.
-
-<br />
 
-## Running the samples
+> [!WARNING]
+> The EOS Overlay is not yet implemented for iOS.
 
-When following the steps to <a href="/readme.md#running-the-samples">run a sample</a> from a build for ios, in Xcode, open the ```.xcodeproj``` from the resulting build folder. Follow the Apple Developer instructions to build and run the app <a href="https://developer.apple.com/documentation/xcode/running-your-app-in-simulator-or-on-a-device">here</a>. If running on a device you may need to <a href="https://developer.apple.com/documentation/xcode/enabling-developer-mode-on-a-device">enable developer mode</a> on the device. This may require you to set up <a href="https://help.apple.com/xcode/mac/current/#/dev80cc24546">automatic signing</a> as well.
-> :heavy_exclamation_mark: Find the build steps in the Unity docs <a href="https://docs.unity3d.com/2021.3/Documentation/Manual/iphone-BuildProcess.html">here</a>.
+## <div align="center">Running the samples</div>
 
-> :heavy_exclamation_mark: Find directions to log in with an Apple ID <a href="/docs/iOS/AppleIDLogin.md">here</a>.
+When following the steps to <a href="/readme.md#running-the-samples">run a sample</a> from a build for ios, in Xcode, open the ```.xcodeproj``` from the resulting build folder. Follow the Apple Developer instructions to build and run the app [here](https://developer.apple.com/documentation/xcode/running-your-app-in-simulator-or-on-a-device). If running on a device you may need to <a href="https://developer.apple.com/documentation/xcode/enabling-developer-mode-on-a-device">enable developer mode</a> on the device. This may require you to set up <a href="https://help.apple.com/xcode/mac/current/#/dev80cc24546">automatic signing</a> as well.
 
-<br />
+> [!INFO]
```

Comment:
This [!INFO] didn't render properly.

Reply:
![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/e6df6ee1-e02b-4208-a42a-f0d4755b2111)